<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
    <title>Jeffrey R. Adrion</title>
    <link
      rel="stylesheet"
      href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600"
    />
    <!-- https://fonts.google.com/specimen/Open+Sans -->
    <link rel="stylesheet" href="css/all.min.css" />
    <!-- https://fontawesome.com/ -->
    <link rel="stylesheet" href="css/bootstrap.min.css" />
    <!-- https://getbootstrap.com/ -->
    <link rel="stylesheet" href="css/tooplate-style.css" />
    <!--
Tooplate 2111 Pro Line
http://www.tooplate.com/view/2111-pro-line
-->
  </head>

  <body>
    <!-- page header -->
    <div class="container" id="home">
      <div class="col-xs-12 text-center">
        <div class="tm-page-header">
             <i class="fas fa-5x fa-chart-bar mr-4"></i>
          <h1 class="d-inline-block text">Jeffrey R. Adrion</h1>
        </div>
      </div>
    </div>
    <!-- navbar -->
    <div class="tm-nav-section">
      <div class="container">
        <div class="row">
          <div class="col-12">
            <nav class="navbar navbar-expand-md navbar-light">
              <button
                class="navbar-toggler"
                type="button"
                data-toggle="collapse"
                data-target="#tmMainNav"
                aria-controls="tmMainNav"
                aria-expanded="false"
                aria-label="Toggle navigation"
              >
                <span class="navbar-toggler-icon"></span>
              </button>

              <div class="collapse navbar-collapse" id="tmMainNav">
                <ul class="navbar-nav mx-auto tm-navbar-nav">
                  <li class="nav-item active">
                    <a class="nav-link" href="#home"
                      >Research<span class="sr-only">(current)</span></a>
                  </li>
                  <li class="nav-item">
                    <a class="nav-link" href="#activities">Software</a>
                  </li>
                  <li class="nav-item">
                    <a class="nav-link external" href="https://scholar.google.com/citations?user=fRxRScsAAAAJ&hl=en&oi=ao" target="_blank">Publications</a>
                  </li>
                  <li class="nav-item">
                    <a class="nav-link external" href="https://www.github.com/jradrion" target="_blank">Github</a>
                  </li>
                  <li class="nav-item">
                    <a class="nav-link" href="#company">About Me</a>
                  </li>
                </ul>
              </div>
            </nav>
          </div>
        </div>
      </div>
    </div>

    
    <!-- Features -->
    <div class="container tm-features-section" id="features">
      <div class="row tm-features-row">
          <!-- <section class="col-md-6 col-sm-12 tm-feature-block"> -->
        <section class="col-xs-12 text-left">
          <header class="tm-feature-header">
            <i class="fas fa-4x fa-dna tm-feature-icon"></i>
            <h2 class="tm-feature-h">My Research</h2>
          </header>
          <p>
          I am a postdoctoral researcher in the
          <a href="http://kernlab.org" target="_blank">Kern Lab</a>
          at the University of Oregon. My work lies at the intersection of population genomics, 
          bioinformatics, and computer science, where I seek to develop computational tools that better 
          allow us to understand how organisms adapt to their environments. 
          </p>
          <p>
          Much of my recent work has focued on using machine learning to infer the genome-wide 
          landscape of recombination rates. Recombination plays an essential role in a myriad 
          of downstream evolutionary and biological processes of key interest to researchers in 
          fields of evolutionary, population, and medical genetics. From the mapping of genetic 
          disease variants in humans, to inferring demographic histories, tools for accurately 
          inferring rates of recombination are indispensable. For decades, population 
          geneticists have shown great interest in developing methods to estimate rates 
          of recombination from genetic data, and while a number of methods are currently in 
          wide use, many suffer from known biases, such as poor performance on small sample 
          sizes or with model misspecification.
          </p>
          <p>
          My work builds on recent breakthroughs in machine leaning technology, in particular the 
          development of deep artificial neural networks, and applies these technologies to the problem of 
          inferring recombination rates. In addition to providing an end-to-end pipeline for 
          inferring rates of recombination that is robust to both small sample sizes and model
          misspecification, I show that genome-wide recombination landscapes are highly correlated 
          among different populations of <i>Drosophila melanogaster</i>, and that differences in recombination rates 
          between populations can in part be attributed to the effects of polymorphic inversions in this species.
          </p>
          <p>
          Machine learning, and deep learning in particular, have the potential to revolutionize the fields of 
          evolutionary and population genomics. My research seeks to be at the forefront of this sea change, 
          and to lead the development of the tools and methods that will accelerate our improved understanding of the 
          adaptive process.
          </p>
        </section>
      </div>
    </div>

    
    <!-- Activities -->
    <div class="container" id="activities">
      <div class="row">
        <div class="col-12">
          <div class="tm-parallax">
            <header class="tm-parallax-header">
              <h2 class="">My Software Projects and Collaborations</h2>
            </header>
          </div>
        </div>
      </div>
      <div class="row">
        <div class="col-lg-6">
          <div class="tm-activity-block">
            <div class="tm-activity-img-container">
              <img src="img/relernn.png" alt="Image" class="tm-activity-img" />
            </div>
            <div class="tm-activity-block-text">
              <h3 class="tm-text-blue">ReLERNN</h3>
              <p>
              A software pipeline that uses a recurrent neural network to infer the 
              genome-wide landscape of recombination rates directly from raw polymorphism 
              data. Phased or unphased VCFs are used as input and modeled without the data 
              compression associated with the use of summary statistics.
              </p>
            </div>
          </div>
        </div>
        <div class="col-lg-6">
          <div class="tm-activity-block mr-lg-0 ml-lg-auto">
            <div class="tm-activity-img-container">
              <img src="img/teflon.png" alt="Image" class="tm-activity-img" />
            </div>
            <div class="tm-activity-block-text">
              <h3 class="tm-text-blue">TEFLoN</h3>
              <p>
              A software pipeline that uses paired-end Illumina sequence data to both discover 
              novel transposable elements (TEs) and perform TE genotyping. Input samples can 
              either be individually sequenced or sequenced as a pool, and multiple samples 
              can be analyzed simultaneously to improve sensitivity.
              </p>
            </div>
          </div>
        </div>
        <div class="col-lg-6">
          <div class="tm-activity-block">
            <div class="tm-activity-img-container">
              <img src="img/sps.png" alt="Image" class="tm-activity-img" />
            </div>
            <div class="tm-activity-block-text">
              <h3 class="tm-text-blue">stdpopsim</h3>
              <p>
              A community-maintained library of standardized population genetic simulation models, spanning
              diverse taxa. These simulations are currently being used to benchmark popular tools aimed at 
              estimating historical population size histories.
              </p>
            </div>
          </div>
        </div>
        <div class="col-lg-6">
          <div class="tm-activity-block">
            <div class="tm-activity-img-container">
              <img src="img/simpoolTE.png" alt="Image" class="tm-activity-img" />
            </div>
            <div class="tm-activity-block-text">
              <h3 class="tm-text-blue">simpoolTE</h3>
              <p>
              A software tool that simulates the insertion and deletion of transposable elements 
              in a population of chromosomes, including the simulation of target site duplications. 
              These simulations can be used to benchmark the accuracy of TE-calling software packages.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>

    <section class="container tm-company-section" id="company">
      <div class="row">
        <!-- <div class="col-xl-9 col-lg-8 col-md-12 tm-company-left"> -->
        <div class="col-xs-4 col-xs-8 tm-company-block">
          <div class="tm-company-about">
            <div class="tm-company-img-container">
              <img src="img/about_me.jpg" alt="Image" />
            </div>
            <div class="tm-company-about-text">
              <header>
                <i class="fas fa-4x fa-signature tm-feature-icon"></i>
                <h2 class="tm-company-about-header">About Me</h2>
              </header>
              <p>
              I got my Ph.D. at Indiana University (IU), where I was broadly trained in evolutionary biology and 
              population genetics. There I worked on a diversity of evolutionary topics, such as characterizing 
              the role of compensatory evolution in driving the evolution of mitochondrial-nuclear gene complexes 
              and identifying how spatially varying selection shapes the genome-wide distribution of 
              transposable elements in <i>Drosophila</i>. At IU I also enrolled in formal coursework to study 
              bioinformatics and machine learning, and my interest in these areas has fueled my desire to develop 
              computational tools to better understand adaptation. 
              <p class="mb-4">
              I currently live in Eugene, Oregon, with my wife, Sarah, and our dog, Sandy. I enjoy camping, 
              biking, and going to see live music. My CV and contact information can be found 
              <a href="http://github.com/jradrion/jradrion.github.io/tree/master/docs/jradrionCV.pdf" target="_blank">here</a>
              .
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
    
    
    <footer class="container tm-footer">
      <div class="row tm-footer-row">
        <p class="col-md-10 col-sm-12 mb-0">
          &copy; 2019 Jeffrey R. Adrion - Design adapted from:
          <a rel="nofollow" href="https://www.facebook.com/tooplate" class="tm-footer-link">Tooplate</a>
        </p>
        <div class="col-md-2 col-sm-12 scrolltop">
          <div class="scroll icon"><i class="fa fa-4x fa-angle-up"></i></div>
        </div>
      </div>
    </footer>

    
    <script src="js/jquery-1.9.1.min.js"></script>
    <!-- Single Page Nav plugin works with this version of jQuery -->
    <script src="js/jquery.singlePageNav.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script>
      /**
       * detect IE
       * returns version of IE or false, if browser is not Internet Explorer
       */
      function detectIE() {
        var ua = window.navigator.userAgent;

        var msie = ua.indexOf("MSIE ");
        if (msie > 0) {
          // IE 10 or older => return version number
          return parseInt(ua.substring(msie + 5, ua.indexOf(".", msie)), 10);
        }

        var trident = ua.indexOf("Trident/");
        if (trident > 0) {
          // IE 11 => return version number
          var rv = ua.indexOf("rv:");
          return parseInt(ua.substring(rv + 3, ua.indexOf(".", rv)), 10);
        }

        // var edge = ua.indexOf('Edge/');
        // if (edge > 0) {
        //     // Edge (IE 12+) => return version number
        //     return parseInt(ua.substring(edge + 5, ua.indexOf('.', edge)), 10);
        // }

        // other browser
        return false;
      }

      function setVideoHeight() {
        const videoRatio = 1920 / 1080;
        const minVideoWidth = 400 * videoRatio;
        let secWidth = 0,
          secHeight = 0;

        secWidth = videoSec.width();
        secHeight = videoSec.height();

        secHeight = secWidth / 2.13;

        if (secHeight > 600) {
          secHeight = 600;
        } else if (secHeight < 400) {
          secHeight = 400;
        }

        if (secWidth > minVideoWidth) {
          $("video").width(secWidth);
        } else {
          $("video").width(minVideoWidth);
        }

        videoSec.height(secHeight);
      }

      // Parallax function
      // https://codepen.io/roborich/pen/wpAsm
      var background_image_parallax = function($object, multiplier) {
        multiplier = typeof multiplier !== "undefined" ? multiplier : 0.5;
        multiplier = 1 - multiplier;
        var $doc = $(document);
        $object.css({ "background-attatchment": "fixed" });
        $(window).scroll(function() {
          var from_top = $doc.scrollTop(),
            bg_css = "center " + multiplier * from_top + "px";
          $object.css({ "background-position": bg_css });
        });
      };

      $(window).scroll(function() {
        if ($(this).scrollTop() > 50) {
          $(".scrolltop:hidden")
            .stop(true, true)
            .fadeIn();
        } else {
          $(".scrolltop")
            .stop(true, true)
            .fadeOut();
        }

        // Make sticky header
        if ($(this).scrollTop() > 158) {
          $(".tm-nav-section").addClass("sticky");
        } else {
          $(".tm-nav-section").removeClass("sticky");
        }
      });

      let videoSec;

      $(function() {
        if (detectIE()) {
          alert(
            "Please use the latest version of Edge, Chrome, or Firefox for best browsing experience."
          );
        }

        const mainNav = $("#tmMainNav");
        mainNav.singlePageNav({
          filter: ":not(.external)",
          offset: $(".tm-nav-section").outerHeight(),
          updateHash: true,
          beforeStart: function() {
            mainNav.removeClass("show");
          }
        });
        
        videoSec = $("#tmVideoSection");

        // Adjust height of video when window is resized
        $(window).resize(function() {
          setVideoHeight();
        });

        setVideoHeight();

        $(window).on("load scroll resize", function() {
          var scrolled = $(this).scrollTop();
          var vidHeight = $("#hero-vid").height();
          var offset = vidHeight * 0.6;
          var scrollSpeed = 0.25;
          var windowWidth = window.innerWidth;

          if (windowWidth < 768) {
            scrollSpeed = 0.1;
            offset = vidHeight * 0.5;
          }

          $("#hero-vid").css(
            "transform",
            "translate3d(-50%, " + -(offset + scrolled * scrollSpeed) + "px, 0)"
          ); // parallax (25% scroll rate)
        });

        // Parallax image background
        background_image_parallax($(".tm-parallax"), 0.4);

        // Back to top
        $(".scroll").click(function() {
          $("html,body").animate(
            { scrollTop: $("#home").offset().top },
            "1000"
          );
          return false;
        });
      });
    </script>
  </body>
</html>
